src/Makefile

```diff
@@ -64,6 +64,9 @@ FLAGS += -DUSE_POLLY
 ifeq ($(USE_POLLY_OPENMP),1)
 FLAGS += -fopenmp
 endif
+ifeq ($(USE_POLLY_ACC),1)
+LLVMLINK += -lPollyPPCG -Wl,-rpath=$(shell $(LLVM_CONFIG_HOST) --libdir),--no-as-needed -lGPURuntime -Wl,--as-needed
```

Comment:
@vtjnash I'm not sure if we can do without `-rpath` since the runtime is a shared library (libGPURuntime.so), therefore,
- cannot be pulled into libjulia.so
- requires us to provide its location in case of external LLVM builds ( for internal builds, all libraries are copied into usr/lib, which is where libjulia.so is located and is covered by the $ORIGIN:$ORIGIN/julia runpath ).

The man page of `ld` in macOS lists `-rpath` as an option,
 ```
 -rpath path

                 Add path to the runpath search path list for image being created.
```

PS :  If I'm not wrong, the -rpath is required to point to an external <llvm_build>/lib when USE_LLVM_SHLIB:=1.